### PR TITLE
REF: Transactions Header JS > TSX

### PR DESCRIPTION
**Pull request by marcosrdz**
Transactions Header has been refactored to be a functional component with hooks and TSX. Please look for bugs

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/wfgxneewmwvvygjkblvgg7fdba?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11109)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/drv2g7m6wlih4ya4lbv2nyrda4?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11110)

File: components/TooltipMenu.ios.js [hunk 1]
```diff
@@ -59,7 +59,13 @@ const ToolTipMenu = (props, ref) => {
       }}
       style={buttonStyle}
     >
-      {props.onPress ? <TouchableOpacity accessibilityRole="button" onPress={props.onPress}>{props.children}</TouchableOpacity> : props.children}
+      {props.onPress ? (
```
Comment (Overtorment) [inline components/TooltipMenu.ios.js:62]: looks like a lot of unnecessary styling changes..?

**Review by Overtorment — APPROVED** [review]
lgtm

File: components/TooltipMenu.ios.js (hunk 1 above)
Comment (marcosrdz) [reply components/TooltipMenu.ios.js:62]: Lint

**Review by ncoelho — APPROVED** [review]

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**